test/extended/util/framework.go

```diff
@@ -2055,3 +2056,11 @@ func GetHypershiftManagementClusterConfigAndNamespace() (string, string, error)
 
 	return hypershiftManagementClusterKubeconfig, hypershiftManagementClusterNamespace, nil
 }
+
+func SkipIfExternalControlplaneTopology(oc *CLI, reason string) {
+	controlPlaneTopology, err := GetControlPlaneTopology(oc)
+	o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
```suggestion
	o.ExpectWithOffset(1,err).NotTo(o.HaveOccurred())
```

that way the failure gets reported as coming from the test in question and not the helper function

https://pkg.go.dev/github.com/onsi/gomega#ExpectWithOffset

Reply:
Updated